### ISSUE TITLE
Button labels

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#147]
The button are not labelled well and causing confusion for the users. 

On the Search screen,

The filled cross icon in the Search field should be labelled clear text button.
Cancel on the right should be labelled as cancel button.
When focused on the search field, it should recognize search field it and when editing the place of insertion point should be specified.
The big cross should be labelled as Close button when using search filters and in web-view.

On the Publications screen,

Notifications label: The notifications icon should be labelled as Notifications button.

On Level-3 the Cross and share button should be properly labelled as close and share button respectively.

**Comment by ForumG-SC** [comment]
Done
